packages/cadl-ranch-specs/http/type/property/optionality/main.tsp

```diff
@@ -128,6 +128,55 @@ model CollectionsModelProperty is ModelTemplate<StringProperty[]>;
 interface CollectionsModel
   extends OperationsTemplate<CollectionsModelProperty, "[{'property': 'hello'}, {'property': 'world'}]"> {}
 
+// Model with optional string literal property
+@doc("Model with string literal property")
+model StringLiteralProperty is ModelTemplate<"hello">;
+@route("/string/literal")
+@operationGroup
+interface StringLiteral extends OperationsTemplate<StringLiteralProperty, "hello"> {}
```

Comment:
```suggestion
interface StringLiteral extends OperationsTemplate<StringLiteralProperty, '"hello"'> {}
```

quote on the string?

Reply:
can't do single quotes in typespec but you can still escape with `\`